docs/topics/con_spring-reactive-introduction.adoc

```diff
@@ -0,0 +1,42 @@
+[id="introduction-to-spring-reactive_{context}"]
+= Introduction to Spring Reactive
+
+The Spring Reactive stack is built on link:https://projectreactor.io/[Project Reactor], a reactive library for building non-blocking applications on the JVM that implements messaging over HTTP based on the the link:https://www.reactive-streams.org/[Reactive Streams specification] and RxJava2.
```

Comment:
JVM that implements messaging over HTTP based on the the: Remove the extra "the".

Reply:
removed :)